addons/core/translations/en-us.yaml

```diff
@@ -50,3 +50,7 @@ themes:
   system: System Default
   light: Light
   dark: Dark
+onboarding:
+  choose-path:
```

Comment:
Can we call this `quick-start` and group everything related to the feature here?

Reply:
Done.